packages/theme-base/css-vars-usage.json

```diff
@@ -151,6 +151,17 @@
   "--sapHC_ReducedBackground",
   "--sapHC_ReducedForeground",
   "--sapHighlightColor",
+  "--sapIllus_AccentColor",
```

Comment:
Please rebuild the project before pushing, just to be sure that these variables are used for real